### ISSUE TITLE
release group parse error

### ISSUE THREAD

**Issue by lightnear** [Radarr/Radarr#7675]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

movie file: [蒙特卡洛].Monte.Carlo.2011.BluRay.720p.x264.AC3-CMCT.mkv
release group: 蒙特卡洛

This release group is wrong. It's chinese name of the movie.

torrent folder and file info:
<img width="1491" alt="image" src="https://user-images.githubusercontent.com/1888302/195800870-3fad2a74-ef00-47a2-bf4c-206ed4377055.png">

### Expected Behavior

correct release group: CMCT

`[{release group}]` and `-{release group}` are parsed as release group. But when both pattern exist, I suggest use `-{release group}` pattern first.

### Environment

```markdown
- OS: Ubuntu 22.04
- Radarr: 4.2.4.6635
- Docker Install: yes
- Using Reverse Proxy: yes
- Browser: Microsoft Edge 106.0.1370.42
- Database: PostgreSQL
```

### What branch are you running?

Master

### Trace Logs?

[radarr.trace.0.txt](https://github.com/Radarr/Radarr/files/9784051/radarr.trace.0.txt)

<img width="1505" alt="image" src="https://user-images.githubusercontent.com/1888302/195801360-28c7b1a9-6d26-43e0-8f51-39dad7244ae0.png">

**Comment by bakerboy448** [comment]
The group named their releases using a format like anime groups where the group is at the beginning of the name

**Comment by lightnear** [comment]
I searched Activity -> History of my radarr, in the early version (maybe 4.0.4), the group is parsed correctly.

In the early version, when grap torrent, the release group is parsed corrently from torrent title (not from movie file name).
while importing, maybe the release group reuse parsed value when grap torrent instead of parsing from movie file name.

The following images are from my radarr Activity -> History, and I don't have logs for them.

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/1888302/196658869-6ad6611f-93dc-4dcf-9f84-214636d9b1db.png">

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/1888302/196659187-46cf17e3-f52d-49e1-8824-9bb8720824ba.png">

**Comment by lightnear** [comment]
I tested on radarr version: **3.2.2.5080**
Release Group can be parsed correctly.

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/1888302/196708763-cd4299e4-13ee-4e83-b65d-b758a8759971.png">
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/1888302/196708875-2762cf39-75fd-4a89-88fb-5956fd764bbd.png">
<img width="1497" alt="image" src="https://user-images.githubuse
rcontent.com/1888302/196709598-ce778a15-d81a-4bda-9a3d-748804683cb8.png">
<img width="1497" alt="image" src="https://user-images.githubusercontent.com/1888302/196711352-20032336-685f-401b-8775-6f3f770b1665.png">


[radarr.trace.txt](https://github.com/Radarr/Radarr/files/9821086/radarr.trace.txt)

**Comment by Qstick** [comment]
Yes, we recently added additional parsing for anime releases which use the group in brackets on the front of the release. Thus it's thinking `[蒙特卡洛]` is the group here.
